pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "prospector"
-version = "1.8.1"
+version = "1.8.3"
```

Comment:
You'll need this as 1.8.2 if you plan to release 1.8.2 :-)

Reply:
I'm going to rebase the first commit is the one I'm going to tag :)